charts/hivemq-operator/README.md

```diff
@@ -2,7 +2,8 @@
 
 See the [documentation](https://docs.hivemq.com/hivemq-operator/) for detailed getting started and configuration guidance.
 
-NOTE: `The HiveMQ Operator (legacy) will be sunsetted and cease to receive updates or further support 6 months after the next LTS release of the HiveMQ Platform. The HiveMQ Platform Operator (new) offers usability, stability, and performance enhancements, and will be the supported version for all future LTS releases.`
+NOTE: `The HiveMQ Operator (legacy) is sunset and will not receive updates or further support.
```

Comment:
```suggestion
NOTE: `The HiveMQ Operator (legacy) is sunset and does not receive updates or further support.
```

Reply:
Question on all these notes: should we include a sunset date as well (April 2025?)